2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -21,8 +21,39 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 
    https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif
 -----------------------------------------------------------------------------*/
+let dancing = false;
 function catWalk() {
-  // TODO complete this function
+  function roundToNearestTen(number) {
+    return Math.floor(number / 10) * 10; // because the width of the screen is
+  }
+  const imgRef = document.querySelector('img');
+  const screenWidth = window.innerWidth;
+  const position = imgRef.style.left;
+  const intPosition = parseInt(position, 10);
+  if (!position) {
+    imgRef.style.left = '0px';
+  }
+  if (intPosition === roundToNearestTen(screenWidth / 2 - 150)) {
+    // 150 = img width/2
```

Comment:
It's best practice to remove commented-out code 🙂 

```suggestion

```

Reply:
Sorry , i put it to not forgot where come the 150 and to make it easier for u too

Reply:
That's okay, no worries! I did not see correctly that it was a comment/explanation. In this case I would recommend to make it a bit more clear that it is a note/explanation, for example by adding 'note' and placing it above the if-statement. You can keep it in though 🙂 

```suggestion
    // Note: 150 = img width/2
```